### ISSUE TITLE
Re-Run Discovery Button Initial Discovery

### ISSUE THREAD

**Issue by BQuinn-Assured** [aws-solutions/workload-discovery-on-aws#511]
**Feature name**
Manual Discovery Kick-Off

**Is your feature request related to a problem? Please describe.**
It is my first time using Workload Discovery and I logged in about an hour ago after full deployment including the Resource Deployments Regional and Global but I am not seeing any workloads discovered. I have no way to know if a discovery is underway or scheduled. I am just told that my Account Role Status is Unknown and Last Discovered is Awaiting Discovery. If I further drill down into the region I get Resources "Not Discovered"  

**Describe the feature you'd like to see implemented**
I want to have the ability to manually kick off a discovery on a per region level and have some visual indicator that a discovery is under way. 
**Describe the value this feature will add to AWS Perspective**
Right Now I do not know if I have a misconfiguration or am simply waiting on the initial discovery

**Describe alternatives you've considered**
An alternative would be to have the status' above set to Initial Discovery In Progress with a time estimate.

**Additional context**
I just don't want to pay for these resources for 24 hours while I hope that everything is properly configured and postpone my troubleshooting and resolution.

**Comment by mrpackethead** [comment]
have a look in the log groups /ecs/workload-discovery-task/  and see if its running. by default it runs every 15 minutes
